### PR TITLE
Tailwindcss Standalone CLI

### PR DESCRIPTION
**Pull request by r3dpoint**
Adding non-official Tailwindcss Standalone CLI devcontainer feature to the registry.

### CODE REVIEW

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
Awesome, thank you! 

Can you help resolve the merge conflicts?

File: _data/collection-index.yml [hunk 1]
```diff
@@ -273,3 +273,9 @@
   contact: https://github.com/ijnek/ros-devcontainer-template/issues
   repository: https://github.com/ijnek/ros-devcontainer-template
   ociReference: ghcr.io/ijnek/ros-devcontainer-template
+- name: Assorted Features
+  maintainer: r3dpoint
+  contact: https://github.com/r3dpoint/devcontainer-features/issues
+  repository: https://github.com/r3dpoint/devcontainer-features
+  ociReference: ghcr.io/r3dpoint/devcontainer-features
+
```
Comment (samruddhikhandale) [inline _data/collection-index.yml]:
```suggestion

```

Nit: remove unwanted white spaces? thank you!

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]
left a minor comment

**Review by samruddhikhandale — APPROVED** [review]

**Review by charlestonXsolutions — COMMENTED** [review]
send it

**Pull request merged**